ROMFS/px4fmu_common/init.d-posix/airframes/4008_gz_advanced_plane

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+#
+# @name Advanced Plane SITL
+#
+
+. ${R}etc/init.d/rc.fw_defaults
+
+PX4_SIMULATOR=${PX4_SIMULATOR:=gz}
+PX4_GZ_WORLD=${PX4_GZ_WORLD:=default}
+PX4_SIM_MODEL=${PX4_SIM_MODEL:=Advanced_Plane}
+
+param set-default SIM_GZ_EN 1
+
+param set-default SENS_EN_GPSSIM 1
+param set-default SENS_EN_BAROSIM 0
```

Comment:
I think the default is already zero, so no need to set the default to zero again